static.html

```diff
@@ -12,7 +12,23 @@
 <body>
   <div id="root">
     <!-- insert your tags here -->
+    <div id="toolbar">
+      <button onclick="turn('left');">TURN-LEFT</button>
+      <button onclick="move();">MOVE</button>
+      <button onclick="turn('right');">TURN-RIGHT</button>
+    </div>
+
+    <div id="board">
+      <table>
+        <tr></tr>
+        <tr></tr>
+        <tr></tr>
+        <tr></tr>
+      </table>
+    </div>
+
   </div>
+  <script src="robot.js"></script>
 </body>
```

Comment:
You were not supposed to include the robot.js file here, because it will do the work that you were asked to do manually. If I take out this script your static web page doesn't match the one shown for index.html.